objects/cluster.go

```diff
@@ -99,15 +96,15 @@ func GetMachine(name, namespace, clusterName, set, version string) clusterv1alph
 				"set":                         set,
 			},
 		},
-		Spec: clusterv1alpha1.MachineSpec{
-			ProviderSpec: clusterv1alpha1.ProviderSpec{},
+		Spec: clusterv1alpha2.MachineSpec{
+			// TODO
```

Comment:
could we get a little more on what is needed here?

Reply:
Unfortunately I cannot give any more guidance here. I believe this entire function will probably have to change to GetDockerMachine instead of GetMachine, but I'm not sure how the UX is going to play out yet... so... TBD?